.github/workflows/benchmark_integer.yml

```diff
@@ -8,6 +8,10 @@ on:
         description: "Run all precisions"
         type: boolean
         default: false
+      run_throughput:
+        description: "Run throughput benchmarks"
+        type: boolean
+        default: false
```

Comment:
as discussed we likely need a way to choose latency and/or run_throughput to have data that's up to date

Reply:
I've implemented a way to do so in `benchmark_signed_integer.yml`. Tell me what you think of it before I spread the logic to the others files.

Reply:
will do